tests/integration/test_meta.py

```diff
@@ -216,3 +214,7 @@ def test_incremental_workflow(program, profile):
 
     # remove a file
     rm_file(file_ids[0], project_id, profile)
+
```

Comment:
@matthewpeterkort yes, added a test here. this will try and create the project a second time